Lib/test/test_posix.py

```diff
@@ -1790,6 +1790,19 @@ def test_dup2(self):
         with open(dupfile) as f:
             self.assertEqual(f.read(), 'hello')
 
+    def test_vfork(self):
+        try:
+            pid = posix.posix_spawn(
```

Comment:
You might use args = self.python_args() and  posix.posix_spawn(args[0], args, ...)